### PR TITLE
Use nightly tf package and include tf benchmarks

### PR DESCRIPTION
**Pull request by vishh**

### CODE REVIEW

**Comment by vishh** [comment]
cc @foxish

File: example/tensorflow-notebook.yaml [hunk 1]
```diff
@@ -47,10 +47,10 @@ spec:
         fsGroup: 0
       containers:
       - name: tf-notebook
-        image: gcr.io/kubeflow/tensorflow-notebook
+        image: gcr.io/kubeflow/tensorflow-notebook-cpu:v1
         resources:
-          limits:
-            nvidia.com/gpu: 1 # Number of GPUs
+#          limits:
```
Comment (foxish) [inline example/tensorflow-notebook.yaml]: Should this just be removed entirely?

Comment (vishh) [reply example/tensorflow-notebook.yaml]: It's there as an example.

**Comment by foxish** [comment]
LGTM

**Pull request merged**